src/lib/github/create-project.js

```diff
@@ -1,12 +1,52 @@
 module.exports = octokit => {
-	return async ({ org, name }) => {
-		/**
-		 * Create a GitHub organisation project.
-		 *
-		 * @see https://octokit.github.io/rest.js/#api-Projects-createForOrg
-		 */
-		const result = await octokit.projects.createForOrg({ org, name });
-
-		return result.data;
+	return async ({ org, repo, user, target, name, description }) => {
+		if (org) {
+			/**
+			 * Create a GitHub organisation project.
+			 *
+			 * @see https://octokit.github.io/rest.js/#api-Projects-createForOrg
+			 */
+			const result = await octokit.projects.createForOrg({
+				org: target,
+				name,
+				body: description
+			});
+
+			return result.data;
+		}
+
+		if (user) {
+			/**
+			 * Create a GitHub user project.
+			 *
+			 * @see https://octokit.github.io/rest.js/#api-Projects-createForAuthenticatedUser
+			 */
+			const result = await octokit.projects.createForAuthenticatedUser({
+				name,
+				body: description
+			});
+
+			return result.data;
+		}
+
+		if (repo) {
+			/**
+			 * Create a GitHub repository project.
+			 *
+			 * @see https://octokit.github.io/rest.js/#api-Projects-createForRepo
+			 */
+			const string = target.split('/');
```

Comment:
```suggestion
			const [owner, repo] = target.split('/');
```

Reply:
As a variable name, `string` is probably best avoided.

Reply:
Yah I wasn't happy with it, but I'm having such a hard time coming up with a better variable name.